plugins/dynamic-plugins-info/src/components/InternalPluginsMap.tsx

```diff
@@ -77,8 +77,8 @@ export const InternalPluginsMap: Record<string, string> = {
     './dynamic-plugins/dist/janus-idp-backstage-plugin-3scale-backend-dynamic',
   '@janus-idp/backstage-plugin-keycloak-backend-dynamic':
     './dynamic-plugins/dist/janus-idp-backstage-plugin-keycloak-backend-dynamic',
-  '@janus-idp/backstage-plugin-argocd':
-    './dynamic-plugins/dist/janus-idp-backstage-plugin-argocd',
+  'backstage-community-plugin-redhat-argocd':
+    './dynamic-plugins/dist/backstage-community-plugin-redhat-argocd',
```

Comment:
I figured that, this entry has to be updated or else `PreInstalled` column will be set to `No` which is wrong.

![Screenshot 2024-09-26 at 9 43 30 AM](https://github.com/user-attachments/assets/41525f20-6f60-45bc-a6c8-f09a396cc1dd)

After updating, it now shows the right value:

![Screenshot 2024-09-26 at 10 24 13 AM](https://github.com/user-attachments/assets/fd4619d2-bccc-494f-9aa5-6aa3e3a69655)